latest/docs-ref-conceptual/install-azure-cli.md

```diff
@@ -24,46 +24,34 @@ For information about the latest release, see the [release notes](release-notes-
 > [!NOTE]
 > If you need the previous version of the Azure CLI, here's how to [install Azure CLI 1.0](/azure/cli-install-nodejs).
 
-## macOS
+## <a name="macOS"/>Install on macOS
 
-> [!WARNING]
-> The Homebrew formula for the Azure CLI, `azure-cli`, is currently out of date and will install a previous version.
-
-1. Install Azure CLI 2.0 with one `curl` command.
+1. Install Azure CLI 2.0 with `curl`.
 
    ```bash
    curl -L https://aka.ms/InstallAzureCli | bash
    ```
 
-2. You may have to restart your command shell for some changes to take effect.
+2. You may have to restart your shell for some changes to take effect.
 
    ```bash
    exec -l $SHELL
    ```
    
 3. Run the CLI from the command prompt with the `az` command.
 
-> [!Note]
-> When you install with InstallAzureCli, [`az component update`](/cli/azure/component#update) isn't supported.
-> To update to the latest CLI, run `curl -L https://aka.ms/InstallAzureCli | bash` again.
-> 
-> To uninstall, see the [manual uninstall instructions](#uninstall).
-
-## Windows
+> [!WARNING]
+> The Homebrew formula for the Azure CLI, `azure-cli`, is currently out of date and will install a previous version.
```

Comment:
Seems a bit harsh to say "ridiculous" 

My original request was to remove this entirely and put it at the bottom. Let's look at the customer experience. We tell them how to use curl, the install it, and have success! Yay, they feel great! Then we tell them that brew is broken. This seems confusing.

Can we move this to the bottom?